src/ResourceManager/Compute/Commands.Compute/Extension/SqlServer/NewAzureVMSqlServerAutoBackupConfig.cs

```diff
@@ -117,6 +120,77 @@ public SecureString StorageKey
             set;
         }
 
+        [Parameter(
+          Mandatory = false,
+          Position = 6,
```

Comment:
@anilyela we ask that there be no more than four positional parameters for a cmdlet. Please remove the position attribute from the parameters you have added to this cmdlet.